tests/distributions/conftest.py

```diff
@@ -204,17 +204,6 @@
                  'test_data': [[0.4], [0.35]]}
             ],
             scipy_arg_fn=lambda loc, scale: ((), {"loc": np.array(loc), "scale": np.array(scale)})),
-    Fixture(pyro_dist=dist.HalfCauchy,
```

Comment:
I suppose we can keep this fixture as is? These tests are fast, and help in catching any errors due to the wrapping layers (e.g. `.expand_by`).

Reply:
Sure, will do. I'll just remove the `loc` argument, which does not exist in the upstream version.